packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -83,12 +86,12 @@ const BrandMessaging = ({
               </div>
             </div>
             <div className="flex-initial max-w-md pr-2">
-              <p className="text-left text-sm font-bold">
+              <p className="text-left text-sm font-bold w-5/6 mobileLandscape:w-full">
                 <FtlMsg id="brand-prelaunch-title">
                   Firefox accounts will be renamed Mozilla accounts on Nov 1
                 </FtlMsg>
               </p>
-              <p className="text-left text-xs">
+              <p className="text-left text-xs w-5/6 mobileLandscape:w-full">
                 <FtlMsg id="brand-prelaunch-subtitle">
                   You’ll still sign in with the same username and password, and
                   there are no other changes to the products that you use.
```

Comment:
nit: the space in front of the "Learn more" link is underlined.
![image](https://github.com/mozilla/fxa/assets/22231637/2c1025e6-13e5-4ade-a5c0-f66492f3a010)

Reply:
Also, this is a separate issue but I checked the `fxa-settings/public/locales/en` after restarting my stack and while I do see the new string for the banners, the `branding.ftl` is not updated to contain the new brand terms. --> Likely the cause of the new brand term not getting pulled into the string in `/settings`, but I'm not sure if this is just a me issue.

Reply:
Oh wow, good catch on the underlined space. 

I found the issue why the string wasn't coming through in settings. See the change to the grunt file. We'd never copy l10n strings from fxa-shared/l10n/branding.ftl which is where these product strings live. Once the translations land, the issue would have gone away, but definitely a pre-existing bug and a good call out!

Reply:
Amazing! Can confirm that the space and brand term are now looking good 👍